spec/support/result_helpers.rb

```diff
@@ -11,6 +11,8 @@ def marc_to_folio_with_stubbed_holdings(marc_record, client: stub_folio_client,
             call_number: (item['a'] || '').strip,
             current_location: item['k'],
             home_location: item['l'],
+            status: 'Available',
+            location: 'GRE-STACKS',
```

Comment:
Will this "wrong" data confuse us later when we're trying to get away from the 999 data? I think it's better to leave it blank or map it from the data we do have.